src/HatchIntakeSubsystem.cpp

```diff
@@ -31,44 +31,24 @@ void HatchIntakeSubsystem::teleop() {
     // Actuates the intake first then changes the diection of the intake wheel based 
     // on which button is pressed, the motor turns off when the hatch is either in 
     // or out depending on the button pressed.
-    // Right Button is spit hatch out and Right Trigger is take hatch in.
+    // Right Button is to outtake the hatch and Right Trigger is take hatch in.
     if (operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON)) {
-        SetHatchActuator(false);
-        SetIntake(false, HatchIn());
+        SetIntake(false, HatchIn(), false);
     }
     if (operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER)) {
-        SetHatchActuator(false);
-        SetIntake(true, !HatchIn());
+        SetIntake(true, !HatchIn(), false);
     }
 
-    // Actuates the intake back up when the button is no longer being pressed
+    // Actuates the intake back up when the button is no longer being pressed and stops the intake.
     if(!(operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::RIGHT_BUTTON) || operatorJoystick->GetButton(CORE::COREJoystick::JoystickButton::RIGHT_TRIGGER))){
```

Comment:
Why do you have an or statement that checks to see if a certain button is not pressed? According to this, if the right button is not being pressed, then the if statement would be executed and the intake motors would be run.